pkgs/test/test/runner/node/runner_test.dart

```diff
@@ -148,8 +157,18 @@ void main() {
         }
       ''').create();
 
-    var test = await runTest(['-p', 'node', '-p', 'vm', 'test.dart']);
-    expect(test.stdout, emitsThrough(contains('+1 -1: Some tests failed.')));
+    var test = await runTest([
+      '-p',
+      'node',
+      '-p',
+      'vm',
+      '-c',
+      'dart2js',
+      '-c',
+      'dart2wasm',
+      'test.dart'
+    ]);
+    expect(test.stdout, emitsThrough(contains('+1 -2: Some tests failed.')));
```

Comment:
Two of these should pass right?

Reply:
I think maybe the environment variable isn't being set for dart2wasm

Reply:
I think it's correct? The test is

```
if (const bool.fromEnvironment("node")) {
  throw TestFailure("oh no");
}
```

so it should _fail_ on node. Previously it was +1/-1 because the configurations were VM and dart2js+node. Now it's +1/-2 because dart2wasm+node as added.

Reply:
Oh, geeze I misread it. Then yes this seems correct :)